tools/cpplint.py

```diff
@@ -4677,7 +4677,7 @@ def _ClassifyInclude(fileinfo, include, is_system):
 
   # Headers with C++ extensions shouldn't be considered C system headers
   if is_system and os.path.splitext(include)[1] in ['.hpp', '.hxx', '.h++']:
-    is_system = False
+      is_system = False
```

Comment:
Indent error (or does this come from upstream?)

Reply:
Google favors is 2 space indentation. There is a pretty good chance that this code was formatted with https://github.com/google/yapf

Reply:
All white-space changes are lifted verbatim from https://github.com/cpplint/cpplint/blob/3d8f6f876dd6e3918e5641483298dbc82e65f358/cpplint.py